gameSource/minitech.h

```diff
@@ -0,0 +1,115 @@
+#ifndef minitech_H
+#define minitech_H
+
+#include "LivingLifePage.h"
+#include <vector>
+#include <string>
+
+
+using namespace std;
```

Comment:
While having `using namespace std;` is "fine" in source files, having it in a header is bad practice and should be removed since it can leak into anything that includes it.

Reply:
Thanks Adk